server/aws/secrets.tf

```diff
@@ -7,6 +11,15 @@ resource "aws_secretsmanager_secret_version" "server_database_url" {
   secret_string = "${var.rds_server_db_user}:${var.rds_server_db_password}@tcp(${aws_rds_cluster.covidshield_server.endpoint})/${var.rds_server_db_name}"
 }
 
+resource "aws_secretsmanager_secret" "server_redis_url" {
+  name = "server-redis-url-${random_string.random.result}"
+}
+
+resource "aws_secretsmanager_secret_version" "server_redis_url" {
+  secret_id     = aws_secretsmanager_secret.server_redis_url.id
+  secret_string = "redis://${aws_elasticache_replication_group.covidshield.primary_endpoint_address}:${aws_elasticache_replication_group.covidshield.port}"
+}
+
```

Comment:
This secret needs to be attached to the following IAM Policies to be available to the services:

```terraform
resource "aws_iam_policy" "covidshield_secrets_manager_key_retrieval"
resource "aws_iam_policy" "covidshield_secrets_manager_key_submission"
```